src/static/css/beng-lod-style.css

```diff
@@ -15,10 +19,13 @@ a:hover {
 
 header {
 	background-color: #0f238c;
+    min-height: 210px;
 }
 
 header .row {
 	min-height: 150px;
+	font-size: 16px;
```

Comment:
Deze is overbodig. Font-sizing nooit meer in px doen, rem only.

Reply:
thanks, font-size nooit meer in px => mental note